public/app/assetlibrary/search/search.html

```diff
@@ -69,7 +69,7 @@
               <option value="" selected>Uploader</option>
             </select>
           </div>
-          <div class="col-sm-4" data-ng-if="sections.length">
+          <div class="col-sm-4" data-ng-show="sections.length">
```

Comment:
Note: Use of `data-ng-if` above frequently setting non-empty 'section' value to empty. The `ng-if` was removing `ng-model` from scope prior to search query. The change above fixes the prob and does not change UX.